### ISSUE TITLE
gtt fails if 'merge requests' feature is off

### ISSUE THREAD

**Issue by daniviga** [kriskbx/gitlab-time-tracker#99]
We have some projects on a self-hosted GitLab instance that are mainly used for issue tracking and thus they have the 'merge requests' feature off.

![Screenshot from 2019-09-27 11-19-12](https://user-images.githubusercontent.com/1818657/65758068-ad08a380-e118-11e9-8c55-1a386dbe5f28.png)

In such cases `gtt` fails with an error:

```
gtt report sysadmin/infrastructure
🔍  Selected projects: sysadmin/infrastructure
🔍  Resolving "sysadmin/infrastructure"... ✓
📦  Fetching issues... ✓
📦  Fetching merge requests... ✗
Error: could not fetch merge requests.
```

Everything works fine with the feature enabled.

Would it be possible to just skip the collection of statistics from 'merge requests' when the feature ha been disabled instead of making the entire process fail?

We are running `gtt` 1.7.40 on Linux.
Many thanks for your work!

**Comment by qelphybox** [comment]
try
```bash
gtt report --query=issues sysadmin/infrastructure 
```
it will skip merge requests

**Comment by qelphybox** [comment]
@daniviga try
```bash
gtt report --query=issues sysadmin/infrastructure
```
it will skip merge requests

**Comment by HacKanCuBa** [comment]
the same happens for me. It should just skip or check before attempting to get MRs if they are enabled.

**Comment by johnjaylward** [comment]
Yes, it would be nice if this could be dynamic since when you are running against a "group" you could have a mix of projects with the features enabled or disabled. Having a "soft fail" if a project is missing issues, or missing merge requests would be optimal
